### PR TITLE
fix(AuctionResults):Display 'bought in' based on boughtIn flag from Metaphysics

### PR DESCRIPTION
**Pull request by bhoggard**
https://artsyproduct.atlassian.net/browse/CX-905

Screen Shot 2021-01-13 at 8 57 39 PM

### CODE REVIEW

File: src/v2/Apps/__tests__/Fixtures/Artist/Routes/AuctionResultsFixture.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const AuctionResultsFixture: AuctionResults_Test_QueryRawResponse = {
             date_text: "1881-1973",
             saleDate: "2020-02-05T19:00:00Z",
             price_realized: { display: "$NaN", cents_usd: 0 },
+            boughtIn: false,
```
Comment (admbtlr) [inline src/v2/Apps/__tests__/Fixtures/Artist/Routes/AuctionResultsFixture.ts:47]: Is there a test for the case were `boughtIn` is `true`?

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Looks good to me 🎉, only comment I have is maybe just adding a test for when `boughtIn` is true

**Comment by bhoggard** [comment]
#squashongreen

**Comment by admbtlr** [comment]
Thanks Barry!

**Pull request merged**